lib/attr_encrypted.rb

```diff
@@ -277,6 +310,7 @@ module InstanceMethods
     #  @user = User.new('some-secret-key')
     #  @user.decrypt(:email, 'SOME_ENCRYPTED_EMAIL_STRING')
     def decrypt(attribute, encrypted_value)
+      self.class.encrypted_attributes[attribute.to_sym][:operation] = :decrypting
```

Comment:
Nevermind, that [`evaluated_options`](https://github.com/attr-encrypted/attr_encrypted/pull/182/files#diff-88607ee1fc6781577944dc73a3630d96R341) hash creates its own copy of the options right after, we're probably good here.